src/module10/Main103.java

```diff
@@ -0,0 +1,15 @@
+package module10;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main103 {
+    public static void main(String[] args) {
+        try {
+            List<Object> list = new ArrayList<>();
+            list.get(0);
+        } catch (Throwable t) {
+            System.out.println("this element does not exist");
+        }
+    }
```

Comment:
Define an object reference and initialize it to null. Try to call a method through this reference.